pkg/apis/externaldns/types.go

```diff
@@ -186,6 +188,8 @@ var defaultConfig = &Config{
 	RFC2136TSIGSecret:           "",
 	RFC2136TSIGSecretAlg:        "",
 	RFC2136TAXFR:                true,
+	TransIPAccountName:          "",
```

Comment:
Just FYI in case you missed it, there's a types_test.go where you can test the behaviour of the configuration for TransIP.